spinegeneric/config/sg_layout_config.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
"sg_" is for CLI scripts. I would call this file `bids_layout_config.json`, and i would rename the specs to `bids_specs.json`. It will be clear from an external user what is the purpose of each of these files.

Reply:
Fixed in https://github.com/spine-generic/spine-generic/pull/198/commits/effbfd9795c732b01ad948fa7d947e2877447607